packages/grafana-prometheus/src/querybuilder/components/MetricCombobox.test.tsx

```diff
@@ -0,0 +1,127 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+
+import '@testing-library/jest-dom';
+import { DataSourceInstanceSettings, MetricFindValue } from '@grafana/data';
+
+import { PrometheusDatasource } from '../../datasource';
+import { PromOptions } from '../../types';
+
+import { MetricCombobox, MetricComboboxProps } from './MetricCombobox';
+
+describe('MetricCombobox', () => {
+  beforeAll(() => {
+    const mockGetBoundingClientRect = jest.fn(() => ({
+      width: 120,
+      height: 120,
+      top: 0,
+      left: 0,
+      bottom: 0,
+      right: 0,
+    }));
+
+    Object.defineProperty(Element.prototype, 'getBoundingClientRect', {
+      value: mockGetBoundingClientRect,
+    });
+  });
```

Comment:
@tskarhed this is needed I think to get the menu to actually take up space in jest's virtual dom? We should find a way to make this easier for consumers to test this with Combobox....

Reply:
Yes, it is in Grafana's Jest setup. I initially introduced a test helper for it, because this will be an issue for every plugin developer doing this. I would prefer some nicer way to automatically solve it though.

Reply:
Maybe the `grafana-prometheus` package isn't using it? 🤔